tweakwcs/imalign.py

```diff
@@ -235,9 +235,9 @@ def fit_wcs(refcat, imcat, tpwcs, ref_tpwcs=None, fitgeom='general', nclip=3,
 
     # check fitgeom:
     fitgeom = fitgeom.lower()
-    if fitgeom not in ['shift', 'rscale', 'general']:
+    if fitgeom not in ['shift', 'rshift', 'rscale', 'general']:
```

Comment:
This list of supported `fitgeom` values is repeated 4 times in this module and elswhere, including  2 docstrings.  Maybe global variable can contain it and be used?